src/components/TextField/TextField.jsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import classNames from 'classnames';
+import _ from 'lodash';
+import { bindClassNames } from '../../util/style-helpers';
+import Validation from '../Validation/Validation';
+
+const boundClassNames = bindClassNames('TextField');
+
+const {
+	bool,
+	string,
+	func,
+	number,
+	object,
+	any,
+} = React.PropTypes;
+
+/**
+ *
+ * {"categories": ["controls", "text"]}
+ *
+ * TextField should cover all your text input needs. It is able to
+ * handle single and multi line inputs.
+ */
+const TextField = React.createClass({
+	propTypes: {
+		/**
+		 * Styles that are passed through to the root `div` container.
+		 */
+		style: object,
+
+		/**
+		 * Styles that are passed through to the underlying `input`.
+		 */
+		inputStyle: object,
```

Comment:
hard to say...you could easily target the native `input` with css, but if you are using all inline styles, this prop helps, but it doesn't go far enough in all cases.